src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+function Button({ title, disabled }) {
+
+    return (
+        <button disabled={disabled} type="button" onClick={() => console.log(title)}>{title}</button>
```

Comment:
Ik zie dit nu ook in de aandachtspunten file. Maar zoals we afgelopen donderdag hebben geleerd kan je een hele functie meegeven als prop. Dat maakt je button nog flexibeler.